inference-engine/src/cldnn_engine/cldnn_infer_request.cpp

```diff
@@ -832,7 +834,9 @@ void CLDNNInferRequest::allocate_outputs() {
         }
         auto blobPtr = create_device_blob(desc, output_layout);
         _deviceOutputs[no.first] = blobPtr;
-        _outputs[no.first] = blobPtr;
+        Blob::Ptr outputBlob = create_host_blob(desc);
```

Comment:
Please add a branch with host blob created on top of cldnn usm_host memory when usm is available to avoid perf regressions.

Reply:
Changed. Now host blob is created using USM pointer, if device blob was created with USM host flags (which is default case on systems with USM support)